android/src/main/java/com/reactlibrary/JudoReactNativePackage.kt

```diff
@@ -0,0 +1,20 @@
+package com.reactlibrary
+
+import com.facebook.react.ReactPackage
+import com.facebook.react.bridge.NativeModule
+import com.facebook.react.bridge.ReactApplicationContext
+import com.facebook.react.uimanager.ViewManager
+import java.util.*
+
+@Suppress("unused")
+class JudoReactNativePackage : ReactPackage {
+    override fun createNativeModules(reactContext: ReactApplicationContext): List<NativeModule> {
+        val modules: MutableList<NativeModule> = ArrayList()
+        modules.add(JudoReactNativeModule(reactContext))
+        return modules
```

Comment:
you can just: `return arrayOf(JudoReactNativeModule(reactContext))`

Reply:
Fixed